### PR TITLE
[TranslationDebugCommand] fixed failing tests.

### PR DESCRIPTION
**Pull request by aitboudad**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Fixed tickets | #14523 |
| Tests pass? | yes |
| License | MIT |

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/Tests/Command/TranslationDebugCommandTest.php [hunk 1]
```diff
@@ -50,7 +50,46 @@ public function testNoDefinedMessages()
...
+        $tester->execute(array('locale' => 'en', 'bundle' => $this->translationDir));
+
+        $this->assertRegExp('/missing/', $tester->getDisplay());
+        $this->assertRegExp('/unused/', $tester->getDisplay());
+    }
+
+    /**
+     * @expectedException \InvalidArgumentException
+     */
+    public function testDebugUnvalidDirectory()
```
Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/Tests/Command/TranslationDebugCommandTest.php]: `testDebugInvalidDirectory()`

File: src/Symfony/Bundle/FrameworkBundle/Tests/Command/TranslationDebugCommandTest.php [hunk 2]
```diff
@@ -50,7 +50,46 @@ public function testNoDefinedMessages()
...
+        $this->assertRegExp('/unused/', $tester->getDisplay());
+    }
+
+    public function testDebugCustomDirectory()
+    {
+        $kernel = $this->getMock('Symfony\Component\HttpKernel\KernelInterface');
+        $kernel->expects($this->once())
+            ->method('getBundle')
+            ->with($this->equalTo($this->translationDir))
+            ->will($this->throwException(new \InvalidArgumentException()));
```
Comment (xabbuh) [inline src/Symfony/Bundle/FrameworkBundle/Tests/Command/TranslationDebugCommandTest.php]: you can use `willThrowException()`

Comment (aitboudad) [inline src/Symfony/Bundle/FrameworkBundle/Tests/Command/TranslationDebugCommandTest.php]: didn't know about it :) thanks

File: src/Symfony/Bundle/FrameworkBundle/Tests/Command/TranslationDebugCommandTest.php (hunk 1 above)
Comment (aitboudad) [inline src/Symfony/Bundle/FrameworkBundle/Tests/Command/TranslationDebugCommandTest.php]: fixed

**Comment by nicolas-grekas** [comment]
:+1:

**Comment by xabbuh** [comment]
:+1:

**Pull request merged**